modules/contact-form/grunion-contact-form.php

```diff
@@ -2038,13 +2038,36 @@ function process_submission() {
 		 * with this work around. */
 		add_filter( 'wp_insert_post_data', array( $plugin, 'insert_feedback_filter' ), 10, 2 );
 
+		$disabled_functions = explode( ',', ini_get( 'disable_functions' ) );
+		
+		// Avoid warnings when print_r() has been disabled on hosting provider for security reasons
+		if ( ! in_array( 'print_r', $disabled_functions ) ) {
+			$all_values_string = print_r( $all_values, true );
+		}
+		// If print_r is disabled, try to use var_export as an alternative solution
+		elseif ( ! in_array( 'var_export', $disabled_functions ) ) {
+			$all_values_string = var_export( $all_values, true );
+		}
+		else {
+			$all_values_string = '';
```

Comment:
If neither `print_r` or `var_export` are available, shouldn't this manually iterate through and build the string itself for searchability, rather than just setting it to empty?